cmd/notation/version.go

```diff
@@ -21,7 +21,7 @@ func versionCommand() *cobra.Command {
 }
 
 func runVersion() {
-	fmt.Printf("Notation: Notary v2, A tool to sign, store, and verify artifacts.\n\n")
+	fmt.Printf("Notation: A tool to sign, store, and verify artifacts.\n\n")
```

Comment:
Suggest removing "store" from the sentence and use the same unified sentence as `Notation - a tool to sign and verify artifacts. `